client/src/components/Suggestions/index.js

```diff
@@ -1,7 +1,12 @@
 import React from "react";
+import { Link } from "react-router-dom";
 
 const Suggestions = props => {
-  const titles = props.results.map(title => <li key={title.id}>{title.title}</li>);
+  const titles = props.results.map(title => (
+    <Link to={"/post/" + title.id}>
+      <li key={title.id}>{title.title}</li>
```

Comment:
the `<li>` should wrap the Link instead of the link text

Reply:
edited it!

Reply:
i got the links to properly work on the bio page now as well.